src/packages/core/src/server.ts

```diff
@@ -33,6 +34,13 @@ import WebsocketServer, { WebSocketCapableFlavor } from "./servers/ws-server";
 import HttpServer from "./servers/http-server";
 import Emittery from "emittery";
 
+// not using the "net" node package in order to avoid having to polyfill this
+// for the browser build.
+// isIPv4 taken from https://github.com/nodejs/node/blob/01323d50c4b24cf730a651d06ba20633905ecbed/lib/internal/net.js#L31
+const v4Seg = "(?:[0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])";
+const IPv4Reg = new RegExp(`^(${v4Seg}[.]){3}${v4Seg}$`);
+const isIPv4 = (s: string) => IPv4Reg.test(s);
```

Comment:
While `server` isn't useful in the browser, our webpack step still processes it. I could have made it ignore the `net` package, or used a polyfill, but this code was so simple I didn't think it was necessary.

I think copy+paste is ultimately better than ignoring or partially polyfilling `net` because if we ever _do_ utilize the `net` package somewhere else the webpack process to succeed, but the actual probably won't work (because the polyfill is incomplete or just missing). Note: polyfilling the entire `net` package isn't possible, since we can't create actual sockets/pipes/etc in the browser.